### ISSUE TITLE
Foxit PDF Reader crash when HideNetworkAdapterMAC=y

### ISSUE THREAD

**Issue by asbai** [sandboxie-plus/Sandboxie#4296]
### Describe what you noticed and did

Foxit PDF Reader crash on startup when `HideNetworkAdapterMAC=y`

### How often did you encounter it so far?

everytime

### Expected behavior

Foxit PDF Reader start normally

### Affected program

Foxit PDF Reader

### Download link

https://foxit-reader.software.informer.com/7.3/

### Where is the program located?

The program is installed only outside the sandbox.

### Did the program or any related process close unexpectedly?

Yes, it did, but no .dmp file has been created in the system.

### What version of Sandboxie are you running now?

5.69.10

### Is it a new installation of Sandboxie?

I just updated Sandboxie from a previous version (I remember which one it is).

### In which sandbox type you have this problem?

In a standard isolation sandbox (yellow sandbox icon).

### Can you reproduce this problem on a new empty sandbox?

Not relevant to my request.

### What is your Windows edition and version?

Win7 x64 64bit

### In which Windows account you have this problem?

I use the built-in Administrator account.

### Please mention any installed security software

none

**Comment by DavidXanatos** [comment]
what other settings did you have set i cant reproduce this one

**Comment by asbai** [comment]
Hi @DavidXanatos here is the minimal reproduceable settings on Win7 x64:

```
[TEST]

Enabled=y
RandomRegUID=y
HideDiskSerialNumber=y
# enable to reproduce the crash
HideNetworkAdapterMAC=y
HideFirmwareInfo=y
#IsBlockCapture=y
ConfigLevel=10
Template=BlockAccessWMI
Template=SkipHook
Template=FileCopy
Template=BlockPorts
Template=LingerPrograms
Template=AutoRecoverIgnore
RecoverFolder=C:\Downloads
BorderColor=#0000FF,ttl
BoxNameTitle=n
NeverDelete=n
NotifyInternetAccessDenied=y
DropAdminRights=y
AutoRecover=y
LingerProcess=foxitprotect.exe
BlockNetworkFiles=y
ClosedFilePath=InternetAccessDevices
ClosedFilePath=\Device\Mup\
# This is the sanboxie container folder
ClosedFilePath=C:\SandboxieContainerFolder\
```

**Comment by DavidXanatos** [comment]
with build 1.15.2 i ccant reproduce the issue on win 7 using your config, so I'll close this, if you encounter the issue with he soon to be released 1.15.2 please re open the issue
